### DISCUSSION TITLE
CD Playback Not Working

### DISCUSSION THREAD

**Discussion by TomFouk** [rern/rAudio#1292] [General]
Hi ! 
I’m new to raudio and I have a problem with it. 
I have a USB CD plug into my Raspberry Pi 3B and when I put a CD in, the interface say that “ID of the CD is not found in the database”. I’ve tried to reboot several time but the problem still here.
Can somebody help me ?

**Comment by Vseason** [comment]
Edit: Switching this feedback to a new bug report, rather than a comment in the general section.
